diff --git a/setup.py b/setup.py
index d757f74..5493a05 100755
--- a/setup.py
+++ b/setup.py
@@ -384,8 +384,7 @@ def kitty_env() -> Env:
         ]
         test_program_src = '''#include <UserNotifications/UserNotifications.h>
         int main(void) { return 0; }\n'''
-        user_notifications_framework = first_successful_compile(
-            ans.cc, '-framework UserNotifications', src=test_program_src, lang='objective-c')
+        user_notifications_framework = '-framework UserNotifications'
         if user_notifications_framework:
             platform_libs.extend(shlex.split(user_notifications_framework))
         else:
